modules/monitoring-granting-user-permissions-using-the-cli.adoc

```diff
@@ -16,17 +21,18 @@ You can grant users permissions to monitor their own projects, by using the Open
 
 .Procedure
 
-* Assign a monitoring role to a user for a project:
+* To assign a monitoring role to a user for a project, enter the following command:
 +
 [source,terminal]
 ----
 $ oc policy add-role-to-user <role> <user> -n <namespace> <1>
```

Comment:
since role is restricted to namespace, the command should be
`oc adm policy add-role-to-user <role> <user> -n <namespace> --role-namespace <namespace> `
see bug https://bugzilla.redhat.com/show_bug.cgi?id=1984904
`namespace` is where the role residents

Reply:
Thank you for letting me know about this!